manual/manual/cmds/top.etex

```diff
@@ -66,9 +66,13 @@ its contents are read as a sequence of OCaml phrases
 and executed as per the "#use" directive
 described in section~\ref{s:toplevel-directives}.
 The evaluation outcode for each phrase are not displayed.
-If the current directory does not contain an ".ocamlinit" file, but
-the user's home directory (environment variable "HOME") does, the
-latter is read and executed as described below.
+If the current directory does not contain an ".ocamlinit" file,
+the file "XDG_CONFIG_HOME/ocaml/init.ml" is looked up according
```

Comment:
I would slightly prefer `$XDG_CONFIG_HOME/ocaml/init.ml`, this makes it clear that it's an environment variable. But I can understand that it would be perhaps confusing in combination with windows environment variables. 

FWIW the XDG specification says: "Specifications may reference this specification by specifying the location of a configuration file as $XDG_CONFIG_DIRS/subdir/filename."

Reply:
> I would slightly prefer `$XDG_CONFIG_HOME/ocaml/init.ml`, this makes it clear that it's an environment variable. 

That would be inconsistent with the way the manual mentions environment variables.

Reply:
Ah, fair enough then.